localstack/services/apigateway/apigateway_listener.py

```diff
@@ -330,12 +331,16 @@ def invoke_rest_api_integration(api_id, stage, integration, method, path, invoca
 
             # apply custom request template
             data_str = data
+            isBase64Encoded = False
             try:
                 data_str = json.dumps(data) if isinstance(data, (dict, list)) else to_str(data)
                 data_str = apply_template(integration, 'request', data_str, path_params=path_params,
                     query_params=query_string_params, headers=headers)
             except Exception:
```

Comment:
I think it would be beneficial to catch the particular error type (e.g., `UnicodeDecodeError`) and then run the mitigation/fallback logic below. In case of a general `Exception`, we can print a `LOG.warning` message and simply `pass` as we did so far.